src/test/java/teammates/test/cases/browsertests/FeedbackRankQuestionUiTest.java

```diff
@@ -176,35 +176,35 @@ public void testInstructorSubmitAndResultsPage() throws Exception {
 
         InstructorFeedbackResultsPage instructorResultsPage =
                 loginToInstructorFeedbackResultsPageWithViewType("instructor1", "instructor", false, "question");
-        clickAjaxLoadedPanelAndWaitForExpansion(instructorResultsPage, "panelHeading-1", "ajax_auto");
+        instructorResultsPage.clickAjaxLoadedPanelAndWaitForExpansion("panelHeading-1", "ajax_auto");
```

Comment:
@damithc `loadPanel("panelHeading-1")` looks good. But for `loatdResultPanel(1)` we have to create different methods as the method sometime used for `"panelHeading-1"` and sometime for `"panelHeading-section-1-2"`
So for which one to go for ?